core/frontend/src/tile/map/MapLayerSettingsService.ts

```diff
@@ -126,7 +126,7 @@ export class MapLayerSettingsService {
     const settingFromName = await IModelApp.settings.getSharedSetting(requestContext, MapLayerSettingsService.SourceNamespace, name, true, projectId, undefined);
     if (settingFromName.setting && storeOnIModel) {
       // SWB What should be done for localization?
-      const errorMessage = IModelApp.i18n.translate("mapLayers:CustomAttach.LayerExistsAsProjectSetting", { layer: settingFromName.setting.name });
+      const errorMessage = IModelApp.i18n.translate("mapLayers:CustomAttach.LayerExistsAsITwinSetting", { layer: settingFromName.setting.name });
```

Comment:
All of these keys will need to updated in `core/frontend/src/public/locales/en/`

Reply:
These keys are under `extensions/map-layer/src/public/locales/en` instead and have been updated.